app/components/avo/fields/belongs_to_field/edit_component.rb

```diff
@@ -63,7 +63,7 @@ def create_path(target_resource = nil)
     helpers.new_resource_path(**{
       via_relation: @field.id.to_s,
       resource: target_resource || @field.target_resource,
-      via_record_id: resource.record.to_param,
+      via_record_id: resource.record.persisted? ? resource.record.to_param : nil,
```

Comment:
Will this hide the link from the `html` template? If not, then what kind of link will that be? How will that record be created if it doesn't have an associated id?